client/client_main.cc

```diff
@@ -110,7 +110,7 @@ void spectator_session_run(const char* server_address, int port) {
   session_loop(&fs, &display, &gpu);
 }
 
-int main(int argc, const char* argv[]) {
+int main(int argc, char* argv[]) {
```

Comment:
note: this change is essentially needed to avoid a `SDL_main` linker error on windows but it also brings it up to spec (ref: [what-are-the-valid-signatures-for-cs-main-function](https://stackoverflow.com/questions/2108192/what-are-the-valid-signatures-for-cs-main-function))